crates/transaction-pool/src/validate/eth.rs

```diff
@@ -957,9 +957,9 @@ impl<Client> EthTransactionValidatorBuilder<Client> {
 
         // TODO: use osaka max blob count once <https://github.com/alloy-rs/alloy/pull/2427> is released
         let max_blob_count = if prague {
```

Comment:
there's one more thing we need to do:

https://github.com/paradigmxyz/reth/blob/55134742d693cebbee1d1cbe19366458a36b3c14/crates/transaction-pool/src/validate/eth.rs#L692-L694